mapbox_tilesets/scripts/cli.py

```diff
@@ -220,7 +220,10 @@ def status(tileset, token=None, indent=None):
 @click.argument("tileset", required=True, type=str)
 @click.option("--token", "-t", required=False, type=str, help="Mapbox access token")
 @click.option("--indent", type=int, default=None, help="Indent for JSON output")
-def tilejson(tileset, token=None, indent=None):
+@click.option(
+    "--secure", "-s", required=False, is_flag=True, help="receive HTTPS resource URLs"
```

Comment:
`-s` is short for `--stage` in the jobs command,  so we should use a different abbreviation here

Reply:
Do the abbreviated versions technically have to be different across commands? I'd also be in favor of _not_ having abbreviated flags.

Reply:
👍 not adding an abbreviated flag here sounds fine to me